src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -1319,17 +1362,38 @@ static int client_receive_message(
 
         case DHCP6_STATE_BOUND:
         case DHCP6_STATE_STOPPED:
+        case DHCP6_STATE_STOPPING:
         default:
                 assert_not_reached();
         }
 
         return 0;
 }
 
+static void client_send_release(sd_dhcp6_client *client) {
+        int r;
+        sd_dhcp6_lease *lease;
```

Comment:
Not a matter, and not a strong rule, but we usually write longer line first:
```
sd_dhcp6_lease *lease;
int r;
```
Also, missing assertion for `client`: `assert(client);`